ceph/rbd/workflows/rbd_trash.py

```diff
@@ -0,0 +1,211 @@
+import json
+
+from ceph.rbd.workflows.rbd import list_images_and_verify_with_input
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def get_trash_list(**kw):
+    """
+    Get all images in trash for a given pool
+    kw:{
+        "rbd": <>,
+        "pool": <>,
+    }
+    """
+    rbd = kw.get("rbd")
+    pool = kw.get("pool")
+    out, err = rbd.trash.ls(pool=pool, format="json")
+    if err:
+        log.error(f"Listing images in trash failed for pool {pool} with err {err}")
+        return []
```

Comment:
why an empty list returning here?

Reply:
Because if success we will be returning list of trash items. If failed we will return empty. Why is this even asked here? Is it any wrong to return empty list?